IFC4x3/PropertyConstants/s/SPRAYED_DRY_3zQuFIQc5D_AZg9HF7zun_.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SPRAYED_DRY_3zQuFIQc5D_AZg9HF7zun_" Name="SPRAYED_DRY" UniqueId="fd6b83d2-6a61-4df8-a8ea-2513c7f78c7e" />
```

Comment:
https://hydrokarst.fr/en/sprayed-concrete/: "a blend of cement and sand, without additional water. The water is added directly at the end of the lance to create the concrete spray. This dry-mix method meets certain spray distance restrictions. Dry-mix offers greater spraying distances than wet-mix. In certain cases, this method also helps manage the concrete preparation and deployment times."

Reply:
Added description in [03a28cc](https://github.com/bSI-InfraRoom/IFC-Specification/pull/824/commits/03a28cc8c6229008d4d538438d999ed34110178f)